audits.yml

```diff
@@ -3,23 +3,38 @@
   hosts: localhost
   gather_facts: false
   tasks:
+    - name: Only gather date_time to facts
+      ansible.builtin.setup:
+        filter:
+          - ansible_date_time
+    - name: Set current date to stats
+      ansible.builtin.set_stats:
+        data:
+          current_date: "{{ ansible_date_time.date }}"
     - name: Get K8S cluster info
       kubernetes.core.k8s_cluster_info:
       register: ocp_version
     - name: Save cluster version
-      ansible.builtin.set_fact:
-        k8s_cluster_version: "{{ ocp_version.version.server.kubernetes }}"
+      ansible.builtin.set_stats:
+        data:
+          k8s_cluster_version: "{{ ocp_version.version.server.kubernetes }}"
     - name: Get OpenShift cluster info
       kubernetes.core.k8s_info:
         api_version: "config.openshift.io/v1"
         kind: ClusterVersion
         name: version
       register: openshift_cluster_version
+      failed_when: openshift_cluster_version.resources[0] not defined
```

Comment:
Hi @bcrochet , this line is the only thing I couldn't put to work. 
In the ansible-navigator it breaks like below:
```
fatal: [localhost]: FAILED! => {"api_found": true, "changed": false, "failed_when_result": "The conditional check 'openshift_cluster_version.resources[0] not defined' failed. The error was: template error while templating string: expected token 'end of statement block', got 'not'.
```
I made it work like this:
`openshift_cluster_version.resources[0] == ""` 

Let me know if you saw something wrong in your tests.

Reply:
Should be fixed.